lint/action.yaml

```diff
@@ -4,27 +4,49 @@ inputs:
   checkout-repo:
     required: false
     description: "Perform checkout as first step of action"
-    default: true
+    default: "true"
   github-token:
     required: true
-    description: "GitHub token that can create/delete comments. Usually - 'secrets.GITHUB_TOKEN'"
+    description: "GitHub token that can checkout the repository. e.g. 'secrets.GITHUB_TOKEN'"
+  commitlint-config-file:
+    required: false
+    description: "Relative or absolute path to the commitlint config file"
+    default: ./.commitlintrc.yml
+  install-gomarkdoc:
+    required: false
+    description: "Whether or not to install gomarkdoc"
+    default: "false"
 runs:
   using: composite
   steps:
-    - uses: actions/checkout@v2
+    - name: Checkout
+      uses: actions/checkout@v2
       if: inputs.checkout-repo
       with:
         fetch-depth: 0
-    - uses: wagoid/commitlint-github-action@v4
+    - name: Setup Tools
+      uses: open-turo/action-setup-tools@v1
+      ## Installs version of golang found in .go-version
+    - name: Authorize
+      uses: open-turo/action-git-auth@v1
```

Comment:
This could be conditionalized if `github-token` is present. I'm not sure all builds will need it.

Reply:
Ok. I will hold off on doing that until it must be conditionalized, but keep this in mind if it does occur.